doc/articles/get-started-vsmac.md

```diff
@@ -3,34 +3,52 @@
 
 ## Prerequisites
-* [**Visual Studio for Mac**](https://visualstudio.microsoft.com/vs/mac/)
+* [**Visual Studio for Mac 8.8**](https://visualstudio.microsoft.com/vs/mac/)
 * [**Xcode**](https://apps.apple.com/us/app/xcode/id497799835?mt=12) 10.0 or higher
 * An [**Apple ID**](https://support.apple.com/en-us/HT204316)
-* .NET Core 3.1
-
-## Modifying Existing Uno App
-
-1. Open project in Visual Studio for Mac
- ![new-project](Assets/quick-start/vs-mac-open-project.png)
-Once open, you should see your folder structure set up like this:
+* [.NET Core SDK 3.1](https://dotnet.microsoft.com/download/dotnet-core/thank-you/sdk-3.1.404-macos-x64-installer)
+* [**GTK+3**](https://formulae.brew.sh/formula/gtk+3) for running the Skia/GTK projects
+
+## Installing the dotnet new templates
+In order to create a new Uno Project, you'll need to open a terminal and run the following:
+```
+dotnet new -i Uno.ProjectTemplates.Dotnet
+```
+Once done, in Visual Studio for Mac, open the preference menu:
+- On left side, open the **Other** then **Preview Features** menu
+- Check the **Show all .NET Core templates in the New Project dialog**
+
+## Create a new project
+1. To create a new project:
+    - Click the **New** button on the welcome screen
+    - Select **Cross Platform App (Uno Platform)**
+    - Choose a name then click create
+
+1. Once created, you should see your folder structure set up like this:
 ![folder-structure](Assets/quick-start/vs-mac-folder-structure.png)\
-If you have a warning symbol on your iOS project, make sure you have the minimum version of XCode.
+    - If you have a warning symbol on your iOS project, make sure you have the minimum version of Xcode installed.
 ![update-xcode](Assets/quick-start/xcode-version-warning.jpg)\
-To update, go to `Visual Studio > Preferences > Projects > SDK Locations > Apple` and select XCode 10.0 or higher.
+To update, go to `Visual Studio > Preferences > Projects > SDK Locations > Apple` and select Xcode 12 or higher.
 Restart Visual Studio.
-
-2. You can now run on iOS, Android, and WebAssembly by setting your startup project and running.
+1. The `dotnet new` template contains WPF projects only compatible with Windows that need to be removed manually:
+    - Select the `MyProject.Skia.Wpf` and `MyProject.Skia.Wpf.Host`
+    - Right click to remove them from the solution.
+    - Note that you can create a `dotnet new` project without those two projects with:
+    ```bash
+    dotnet new unoapp -o MyProject --skia-wpf=false
+    ```
+1. You can now run on iOS, Android, and macOS by setting your startup project and running.
 ![startup-projects](Assets/quick-start/vs-mac-build.png)
    
-Note: You will not be able to build the UWP project on a Mac. All changes to this project must be made on Windows.
+Note: You will not be able to build the UWP and WPF projects on a Mac. All changes to this project must be made on Windows.
```

Comment:
This is confusing, you've just told us to delete the WPF head. 

Also - why does the WPF head need to be deleted, but the UWP head not?

Reply:
Fair question, this section may need to be updated with the updated package I'm about to test (Where it's possible to remove items based on the IDE).

Reply:
The problem here will be removed soon-ish one VS updates. I'll keep it as a heads up for people that want to still do windows dev.